src/datasets/arrow_dataset.py

```diff
@@ -2226,7 +2238,7 @@ def apply_function_on_filtered_inputs(inputs, indices, check_same_num_examples=F
                 additional_args += (effective_indices,)
             if with_rank:
                 additional_args += (rank,)
-            processed_inputs = function(*fn_args, *additional_args, **fn_kwargs)
+            processed_inputs = function(*fn_args, *additional_args, **fn_kwargs, **tqdm_kwargs)
```

Comment:
I don't think this is the right place to pass the `tqdm_kwargs `.

You probably want to pass it to `pbar = utils.tqdm(...)` a few lines later instead